packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -118,15 +118,30 @@
 		}
 	}
 
-	.block-editor-block-toolbar {
```

Comment:
I accidentally removed this CSS in the previous commit. I’ll add it back in the next commit along with any other changes (if needed).